Celestia/celestia-starter/project.ts

```diff
@@ -0,0 +1,93 @@
+import {
+  CosmosDatasourceKind,
+  CosmosHandlerKind,
+  CosmosProject,
+} from "@subql/types-cosmos";
+
+// Can expand the Datasource processor types via the genreic param
+const project: CosmosProject = {
+  specVersion: "1.0.0",
+  version: "0.0.1",
+  name: "Celestia-starter",
+  description:
+    "This project can be use as a starting point for developing your Cosmos Celestia based SubQuery project",
+  runner: {
+    node: {
+      name: "@subql/node-cosmos",
+      version: ">=3.0.0",
+    },
+    query: {
+      name: "@subql/query",
+      version: "*",
+    },
+  },
+  schema: {
+    file: "./schema.graphql",
+  },
+  network: {
+    /* The genesis hash of the network (hash of block 0) */
+    chainId: "mocha-4",
+    /**
+     * These endpoint(s) should be non-pruned archive nodes
+     * Public nodes may be rate limited, which can affect indexing speed
+     * When developing your project we suggest getting a private API key
+     * We suggest providing an array of endpoints for increased speed and reliability
+     */
+    endpoint: ["http://65.108.238.61:13657"],
```

Comment:
Who's endpoint is this, you should look at public endpoints in https://github.com/cosmos/chain-registry/blob/master/celestia/chain.json